docs/source/usage.rst

```diff
@@ -233,3 +233,75 @@ Settings related to the S3 bucket
       "aws_region": "eu-central-1",
       "bucket_name": "amo-blocked-pages"
     }
+
+How to create the ``validate_signature`` function on Amazon Lambda?
```

Comment:
Mega-nit: I'd probably write `How do I create ...?`, which is how this question would get asked in conversational English.